test.tf

```diff
@@ -108,24 +123,99 @@ resource "azurerm_postgresql_server" "example" {
-  value               = "off"
+  value               = "on"
+}
+data "azurerm_client_config" "example_clientconfig" {}
+resource "azurerm_postgresql_active_directory_administrator" "example_adconfig" {
+  server_name         = azurerm_postgresql_server.example.name
+  resource_group_name = azurerm_resource_group.resource_group.name
+  login               = "sqladmin"
+  tenant_id           = data.azurerm_client_config.example_clientconfig.tenant_id
+  object_id           = data.azurerm_client_config.example_clientconfig.object_id
+}
+data "azurerm_client_config" "example_clientconfig" {}
+resource "azurerm_postgresql_active_directory_administrator" "example_adconfig" {
+  server_name         = azurerm_postgresql_server.example.name
+  resource_group_name = azurerm_resource_group.resource_group.name
+  login               = "sqladmin"
+  tenant_id           = data.azurerm_client_config.example_clientconfig.tenant_id
+  object_id           = data.azurerm_client_config.example_clientconfig.object_id
+}
+data "azurerm_client_config" "example_clientconfig" {}
+resource "azurerm_mysql_active_directory_administrator" "example_adconfig" {
+  server_name         = azurerm_mysql_server.example.name
+  resource_group_name = azurerm_resource_group.resource_group.name
+  login               = "sqladmin"
+  tenant_id           = data.azurerm_client_config.example_clientconfig.tenant_id
+  object_id           = data.azurerm_client_config.example_clientconfig.object_id
+}
+data "azurerm_client_config" "example_clientconfig" {}
+resource "azurerm_mysql_active_directory_administrator" "example_adconfig" {
+  server_name         = azurerm_mysql_server.example.name
+  resource_group_name = azurerm_resource_group.resource_group.name
+  login               = "sqladmin"
+  tenant_id           = data.azurerm_client_config.example_clientconfig.tenant_id
+  object_id           = data.azurerm_client_config.example_clientconfig.object_id
+}
+resource "azurerm_storage_account" "example" {
+  name                     = "accteststorageaccount"
+  resource_group_name      = azurerm_resource_group.example.name
+  location                 = azurerm_resource_group.example.location
+  account_tier             = "Standard"
+  account_replication_type = "GRS"
+}
+resource "azurerm_storage_container" "example" {
+  name                  = "accteststoragecontainer"
+  storage_account_name  = azurerm_storage_account.example.name
+  container_access_type = "private"
+}
+resource "azurerm_mssql_server_security_alert_policy" "example" {
+  resource_group_name = azurerm_resource_group.example.name
+  server_name         = server.config.name
+  state               = "Enabled"
+}
+resource "azurerm_mssql_server_vulnerability_assessment" "example" {
+  server_security_alert_policy_id = azurerm_mssql_server_security_alert_policy.example.id
+  storage_container_path          = "${azurerm_storage_account.example.primary_blob_endpoint}${azurerm_storage_container.example.name}/"
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: Ensure ‘log_checkpoints’ are enabled for Azure PostgreSQL Configuration**
 
**Vulnerability**
Server parameter 'log_checkpoints' is not set to 'ON' for Azure PostgreSQL Configuration, this may make audit challenging.
 
**Remediation**
Azure Configuration for PostgreSQL allows you to configure and access Azure PostgreSQL Configuration's standard logs. The logs can be used to identify, troubleshoot, and repair configuration errors and suboptimal performance. Logging information you can configure and access includes errors, query information, auto-vacuum records, connections, and checkpoints.

source: accurics-beta